Makefile

```diff
@@ -1,6 +1,6 @@
 local-webpage :
 	cp -rf README.md docs
-	docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:6.2.4
+	docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material:6.2.4 -- build
```

Comment:
Actually generating local the site instead of binding to a port is not easy to test local changes like check if the docs are good or how the UI behave, but I can change it later